src/devices/tuya.ts

```diff
@@ -811,6 +811,63 @@ const definitions: DefinitionWithExtend[] = [
         toZigbee: [],
         exposes: [e.gas(), e.tamper()],
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: 'Melery',
```

Comment:
This manufaturerName is unusual, could you provide the data/database.db entry of this device?